src/utils/compileMDX.ts

```diff
@@ -60,13 +60,15 @@ export default async function compileMDX(
       .join('\n');
 
   // Turn the MDX we just read into some JS we can execute.
-  const {remarkPlugins} = require('../../plugins/markdownToHtml');
+  // const {remarkPlugins} = require('../../plugins/markdownToHtml');
```

Comment:
check why we aren't able to use remark / rehype plugins after upgrading it.

getting 

```
Error: Expected usable value but received an empty preset, which is probably a mistake: presets typically come with `plugins` and sometimes with `settings`, but this has neither
```